lightning_hpo/commands/logs/show.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import sys
+from typing import List
+
+from lightning.app.cli.commands.connection import _retrieve_connection_to_an_app
+from lightning.app.cli.commands.logs import _show_logs
+from lightning.app.utilities.commands import ClientCommand
+from pydantic import BaseModel
+
+
+class ShowLogsConfig(BaseModel):
+    name: str
+    components: List[str]
+
+
+class ShowLogsCommand(ClientCommand):
+
+    description: str = "Show the logs of an Experiment or a Sweep."
+
+    def run(self):
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            "--name", required=True, type=str, help="The name of the experiment or Sweep to show logs from."
+        )
+        parser.add_argument("-f", "--follow", action="store_true", default=False)
+        hparams = parser.parse_args()
+
+        logs_config: List[ShowLogsConfig] = self.invoke_handler()
+
+        components = []
+        sweeps = [c["name"] for c in logs_config if len(c["components"]) > 1]
+        experiments = [c["name"] for c in logs_config if len(c["components"]) == 1]
```

Comment:
I'm not sure I understand this, isn't it getting names twice from the same collection?

Reply:
This won't happen. You can enter either a sweep name or experiment name. So, there won't be collision.

Reply:
Unless you want to enable users to pass a collection of names. In this case, we simply add list(set(components))

Reply:
That's right, but then the message below 
```
f"The provided name {hparams.name} wasn't found."
f" Here are the sweeps {sweeps} and experiments {experiments}."
```
needs to be fixed.

Reply:
It should be all good. We are filtering experiments from sweeps, so the error message should be good.